README.md

```diff
@@ -1,5 +1,110 @@
 # qlik-cli
 
-This repo is a placeholder for a tool that will be open-sourced in the near future.
+**qlik-cli**, known on the command line simply as qlik, is a CLI generated directly from API specifications. It is meant to be used for basically any task against Qlik Sense SaaS.
+
```

Comment:
We should also mention that it includes all the public APIs of the Qlik Sense SaaS offering.